### ISSUE TITLE
Silo inspector suggestion

### ISSUE THREAD

**Issue by judg393** [jtsage/FS22_ProductionInspector#15]
I know I know I've not fixed the last thing I broke but I wonder if it would be possible to find a way to block silo inspector showing contents of a silo that silo can't take/deliver. Eg I have a mf silo right next to my hayloft. I don't want to see my MF silo contents in my hayloft cos I can't drive to my hayloft and get it from there. 

Probably a me issue  and maybe or limited wish for anyone else

**Comment by jtsage** [comment]
Can you send me a screen cap?  I think this one might not be on me, it might be related to the vagueness of how the storage system actually works (in particular, silo extensions are kind wacky)

**Comment by judg393** [comment]
![image](https://user-images.githubusercontent.com/55981682/217359708-a0a93f2d-3d29-49b4-bc4e-970676d0304d.jpeg)


Note the working PI 👍 and the fact my lizard diesel tank has sugar beet in .... 😂

**Comment by judg393** [comment]
I don't appear to be able to instantly grab sCreen grabs from steam w/out quit and restart I'll try and update you with a screen grab later

**Comment by jtsage** [comment]
Interesting.  I assume the sugar beets and the potatoes are *actually* in the Calk FarmSilo?

**Comment by judg393** [comment]
Yes. well I will try the diesel tank but I had the same thing in the bale storage and it wouldn't let me get the "crop" (potato iirc )from it

**Comment by judg393** [comment]
Look proper Screenie :) 
![image](https://user-images.githubusercontent.com/55981682/217643847-b9cb2056-b7ab-43d1-b95c-6e425a907df4.jpeg)
